configs/permissions/vulnerability.json

```diff
@@ -1,3 +1,10 @@
 {
-    "*": ["*"]
+    "*": ["*", "read", "write"],
+    "vulnerability_results": ["read"],
+    "cve.business_risk": ["write"],
+    "cve.status": ["write"],
+    "cve_system.cve.status": ["write"],
+    "report": ["read", "write"],
+    "system": ["exclude", "include"],
```

Comment:
per our discussion - operations to filter systems account wide via "include" and "exclude" should be combined into a single "write" permission against the opt_out system component e.g. vulnerability:system.exclude:write

Reply:
Replaced by vulnerability:system:write